module/Photo/src/Photo/Service/Photo.php

```diff
@@ -193,6 +139,19 @@ public function getPreviousPhoto($photo)
     }
 
     /**
+     * Get all photos in an album
+     * 
+     * @param Photo\Model\Album $album the album to get the photos from
+     * @param integer $start the result to start at
+     * @param integer $max_results max amount of results to return, null for infinite
+     * @return array of Photo\Model\Album
+     */
+    public function getPhotos($album, $start = 0, $max_results = null)
```

Comment:
max_results moet hernoemd worden naar maxResults om consistent te blijven met de naamgeving in de rest van de applicatie